modules/perception/common/cuda_conf.cc

```diff
@@ -0,0 +1,42 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#include "modules/perception/common/cuda_conf.h"
+
+
+namespace apollo {
+namespace perception {
+
+CudaConfig::CudaConfig(): block_width_(32)
+    , block_height_(32) {
+}
```

Comment:
Use const static variable for 32.

Reply:
OK, I will modify this issue. Thanks for your proposal.